cmd/soroban-cli/src/commands/keys/generate.rs

```diff
@@ -52,29 +57,47 @@ pub struct Cmd {
     /// Fund generated key pair
     #[arg(long, default_value = "false")]
     pub fund: bool,
+
+    /// Overwrite existing identity if it already exists.
+    #[arg(long)]
+    pub overwrite: bool,
 }
 
 impl Cmd {
     pub async fn run(&self, global_args: &global::Args) -> Result<(), Error> {
+        let print = Print::new(global_args.quiet);
+
+        if self.config_locator.read_identity(&self.name).is_ok() {
+            if !self.overwrite {
+                return Err(Error::IdentityAlreadyExists(self.name.clone()));
+            }
+
+            print.infoln(format!("Overwriting identity '{}'", &self.name));
```

Comment:
Instead of the info symbol, maybe we need an ❗ to signify destruction. Wdyt?

Reply:
I think it's overkill given that you already provided `--overwrite` explicitly. But I'll do it anyway.

Reply:
That's a good point. And we should probably reserve red icons like ❗ for errors given the red => error signal.

Reply:
We do have `error` and `errorln` with ❌ though (so we're good).